src/modules/course-list/components/course-card/CourseCard.component.tsx

```diff
@@ -2,18 +2,18 @@ import { FC, useEffect, useRef, useState } from 'react';
 import { FaStar } from 'react-icons/fa';
 import { useNavigate } from 'react-router-dom';
 
-import { CourseItemPreview } from '../../@types/types';
-import { Button } from '../common/UI/button/Button.component';
+import { CourseItemPreview } from '../../../../shared/@types/types';
+import { Button } from '../../../../shared/UI/button/Button.component';
+import { VideoPlayer } from '../../../../shared/components/video-player/VideoPlayer.component';
 import { SkillsList } from '../skills-list/SkillsList.component';
-import { VideoPlayer } from '../video-player/VideoPlayer.component';
 
-import './CourseItem.scss';
+import './CourseCard.scss';
```

Comment:
From the point of clean code principles, you need to specify for what component this interface is used. E.g: `ICourseCardProps`